composer.json

```diff
@@ -22,7 +22,8 @@
     "require": {
         "php": ">=5.3.2",
         "symfony/framework-bundle": ">=2.1",
-        "symfony/monolog-bundle": "~2.3"
+        "symfony/monolog-bundle": "~2.3",
+        "guzzlehttp/guzzle": "~5.2"
```

Comment:
I'm not sure how this will conflict with other Guzzle implementation and how it else can be solved. Input needed!